internal/files/copy.go

```diff
@@ -7,22 +7,24 @@ package files
 import (
 	"os"
 	"path/filepath"
+	"slices"
+	"strings"
 
 	"github.com/magefile/mage/sh"
 )
 
 // CopyAll method copies files from the source to the destination skipping empty directories.
 func CopyAll(sourcePath, destinationPath string) error {
-	return CopyWithSkipped(sourcePath, destinationPath, []string{})
+	return CopyWithSkipped(sourcePath, destinationPath, []string{}, []string{})
 }
 
 // CopyWithoutDev method copies files from the source to the destination, but skips _dev directories and empty folders.
 func CopyWithoutDev(sourcePath, destinationPath string) error {
-	return CopyWithSkipped(sourcePath, destinationPath, []string{"_dev", "build", ".git"})
+	return CopyWithSkipped(sourcePath, destinationPath, []string{"_dev", "build", ".git", ".DS_Store"}, []string{".swp"})
```

Comment:
Added `.DS_Store` as another folder to skip, and a new parameter to skip hidden files based on their extension.